ui/page/odyseeMembership/view.jsx

```diff
@@ -0,0 +1,613 @@
+      return 'All Premium features, and No Ads';
+    }
+  }
+
+  if (!stillWaitingFromBackend && planValue && cardSaved) {
+    setTimeout(function () {
+      // clear query params
+      window.history.replaceState(null, null, window.location.pathname);
+
+      setHasShownModal(true);
+
+      // open confirm purchase
+      // $FlowFixMe
+      document.querySelector('[plan="' + plan + '"][interval="' + interval + '"]').click();
+    }, timeoutValue);
+  }
+
+  const helpText = (
+    <div className="section__subtitle">
+      <p>
+        {__(
+          `Create a Livestream by first submitting your livestream details and waiting for approval confirmation. This can be done well in advance and will take a few minutes.`
+        )}{' '}
+        {__(
+          `Scheduled livestreams will appear at the top of your channel page and for your followers. Regular livestreams will only appear once you are actually live.`
+        )}{' '}
+        {__(
+          `Once the your livestream is confirmed, configure your streaming software (OBS, Restream, etc) and input the server URL along with the stream key in it.`
+        )}
+      </p>
+    </div>
+  );
+
+  return (
+    <>
+      <Page>
+        {/** splash frontend **/}
+        {!stillWaitingFromBackend && purchasedMemberships.length === 0 && !planValue && !hasShownModal ? (
+          <MembershipSplash pageLocation={'confirmPage'} currencyToUse={currencyToUse} />
+        ) : (
+          /** odysee membership page **/
+          <div className={'card-stack'}>
+            {!stillWaitingFromBackend && cardSaved !== false && (
+              <>
+                <h1 style={{ fontSize: '23px' }}>Odysee Premium</h1>
+                {/* let user switch channel */}
+                <div style={{ marginTop: '10px' }}>
+                  <ChannelSelector uri={activeChannelClaim && activeChannelClaim.permanent_url} />
+
+                  {/* explainer help text */}
+                  <Card
+                    titleActions={
+                      <Button
+                        button="close"
+                        icon={showHelp ? ICONS.UP : ICONS.DOWN}
+                        onClick={() => setShowHelp(!showHelp)}
+                      />
+                    }
+                    title={__('Get More Information')}
+                    subtitle={<>{__(`Learn more about how Odysee Premium will work`)} </>}
+                    actions={showHelp && helpText}
+                    className={'premium-explanation-text'}
+                  />
+                </div>
+              </>
+            )}
+
+            {/** available memberships **/}
+            {/* if they have a card and don't have a membership yet */}
+            {!stillWaitingFromBackend && membershipOptions && purchasedMemberships.length < 1 && cardSaved !== false && (
+              <>
+                <div className="card__title-section">
+                  <h2 className="card__title">Available Memberships</h2>
```

Comment:
This entire component lacks i18n handling.
Don't want to block release just because of this, but plus file an issue to handle it someday.